/debugstore kills ultralight node

### Context
1.18.2-759f56f7

### Summary
Queried the /debugstore API on an ultralight node and it paniced:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa9ec14]

goroutine 138397 [running]:
github.com/ethersphere/bee/pkg/storer/internal/reserve.(*Reserve).Radius(...)
        github.com/ethersphere/bee/pkg/storer/internal/reserve/reserve.go:438
github.com/ethersphere/bee/pkg/storer.(*DB).DebugInfo.func4()
        github.com/ethersphere/bee/pkg/storer/debug.go:123 +0x24
golang.org/x/sync/errgroup.(*Group).Go.func1()
        golang.org/x/sync@v0.3.0/errgroup/errgroup.go:75 +0x58
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 138407
        golang.org/x/sync@v0.3.0/errgroup/errgroup.go:72 +0x98
```

### Expected behavior
I would expect the valid parts of the /debugstore to be successfully returned.   No API should be able to panic a running bee node of any mode.

### Actual behavior
panic stack trace is provided above

### Steps to reproduce
Just do a /debugstore against an ultralight (or probably even light) node.

### Possible solution
Don't try to query the stores that aren't initialized in the node.  Pretty crazy place to panic though, querying the radius.
https://github.com/ethersphere/bee/blob/759f56f7d71a76edcfb3122ef044490306d4e6b6/pkg/storer/internal/reserve/reserve.go#L438
https://github.com/ethersphere/bee/blob/759f56f7d71a76edcfb3122ef044490306d4e6b6/pkg/storer/debug.go#L123

--- a/pkg/storer/debug.go
+++ b/pkg/storer/debug.go
@@ -54,7 +54,10 @@ type Info struct {
 func (db *DB) DebugInfo(ctx context.Context) (Info, error) {
 	eg, ctx := errgroup.WithContext(ctx)
 
-	totalChunks, sharedSlots := 0, 0
+	var (
+		totalChunks int
+		sharedSlots int
+	)
 	eg.Go(func() error {
 		return chunkstore.IterateChunkEntries(
 			db.repo.IndexStore(),
@@ -76,7 +79,10 @@ func (db *DB) DebugInfo(ctx context.Context) (Info, error) {
 		)
 	})
 
-	uploaded, synced := 0, 0
+	var (
+		uploaded int
+		synced   int
+	)
 	eg.Go(func() error {
 		return upload.IterateAll(
 			db.repo.IndexStore(),
@@ -98,7 +104,10 @@ func (db *DB) DebugInfo(ctx context.Context) (Info, error) {
 		)
 	})
 
-	collections, chunkCount := 0, 0
+	var (
+		collections int
+		chunkCount  int
+	)
 	eg.Go(func() error {
 		return pinstore.IterateCollectionStats(
 			db.repo.IndexStore(),
@@ -118,29 +127,37 @@ func (db *DB) DebugInfo(ctx context.Context) (Info, error) {
 		)
 	})
 
-	reserveSizeWithinRadius := 0
-	eg.Go(func() error {
-		return db.reserve.IterateChunksItems(db.repo, db.reserve.Radius(), func(ci reserve.ChunkItem) (bool, error) {
-			reserveSizeWithinRadius++
-			return false, nil
-		})
-	})
+	var (
+		reserveCapacity         int
+		reserveSize             int
+		reserveSizeWithinRadius int
 
-	cacheSize, cacheCapacity := db.cacheObj.Size(), db.cacheObj.Capacity()
-
-	reserveSize, reserveCapacity := 0, 0
+		lastBinIDs []uint64
+		epoch      uint64
+	)
 	if db.reserve != nil {
-		reserveSize, reserveCapacity = db.reserve.Size(), db.reserve.Capacity()
+		reserveCapacity = db.reserve.Capacity()
+		reserveSize = db.reserve.Size()
+		eg.Go(func() error {
+			return db.reserve.IterateChunksItems(db.repo, db.reserve.Radius(), func(ci reserve.ChunkItem) (bool, error) {
+				reserveSizeWithinRadius++
+				return false, nil
+			})
+		})
+
+		var err error
+		lastBinIDs, epoch, err = db.ReserveLastBinIDs()
+		if err != nil {
+			return Info{}, err
+		}
 	}
 
 	if err := eg.Wait(); err != nil {
 		return Info{}, err
 	}
 
-	lastbinIds, epoch, err := db.ReserveLastBinIDs()
-	if err != nil {
-		return Info{}, err
-	}
+	cacheSize := db.cacheObj.Size()
+	cacheCapacity := db.cacheObj.Capacity()
 
 	return Info{
 		Upload: UploadStat{
@@ -159,7 +176,7 @@ func (db *DB) DebugInfo(ctx context.Context) (Info, error) {
 			SizeWithinRadius: reserveSizeWithinRadius,
 			TotalSize:        reserveSize,
 			Capacity:         reserveCapacity,
-			LastBinIDs:       lastbinIds,
+			LastBinIDs:       lastBinIDs,
 			Epoch:            epoch,
 		},
 		ChunkStore: ChunkStoreStat{

